frontend/components/panels/HighestDebtBidCheckPanel.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title> {{ currentStateAndTitle.title }} </template>
+        <TextBlock v-if="isExplanationsShown">
+            In order to win the auction, participants can bid on the auction. In case someone else puts the bid with
+            the lowest amount of MKR to receive, you receive your DAI back, only loosing transaction fee.
```

Comment:
Let's change the wording here to: 
"In order to win the auction, participants can bid on the auction. In case someone else puts the bid with a lower compensation, you receive your DAI back, only loosing transaction fee."

Reply:
I added a "the" in front of the last "transaction fee", otherwise the texts are changed!